helmchart/sdp/values.yaml

```
@@ -252,40 +348,65 @@ anchorPlatform:
     tls:
       - hosts:
           - '{{ include "sdp.ap.domain" . }}'
-        secretName: backend-tls-cert-name    # You need to create this secret manually. For more instructions, plz refer to helmchart/docs/README.md
+        secretName: backend-tls-cert-name
         # NOTE: the hosts to be used here will be the same ones as in the anchorPlatform.route section.
 
 
 # =========================== START tss ===========================
+## @section Transaction Submission Service
+## @descriptionStart
+## Configuration parameters for the Transaction Submission Service. This is the service that submits all payment transactions to the Stellar network.
+##  This service is designed to maximize payment throughput, handle queuing, and graceful resubmission/error handling
+## @descriptionEnd
 tss:
-  # =========================== START tss.route ===========================
+  ## @extra tss.route Configuration related to the routing of the TSS.
+  ## @param tss.route.schema Protocol scheme used for the service. Can be "http" or "https".
+  ## @param tss.route.port Primary port on which the TSS listens.
+  ## @param tss.route.metricsPort Port dedicated to metrics collection for the TSS.
   route:
     schema: "https"
     port: "9000"
     metricsPort: "9002"
-  # =========================== START tss.deployment ===========================
+
+
+  ## @extra tss.deployment Configuration related to the deployment of the TSS.
+  ## @param tss.deployment.annotations Annotations to be added to the deployment.
+  ## @extra tss.deployment.podAnnotations Annotations specific to the pods, mainly for integration with monitoring tools like Prometheus.
+  ## @param tss.deployment.podAnnotations.prometheus.io/path The path where Prometheus metrics are exposed.
+  ## @param tss.deployment.podAnnotations.prometheus.io/port The port on which Prometheus metrics are exposed. This should match the TSS metrics port.
+  ## @param tss.deployment.podAnnotations.prometheus.io/scrape Indicates if Prometheus should scrape metrics from this pod.
+  ## @extra tss.deployment.strategy Configuration related to the deployment strategy, ensuring smooth updates and minimal downtime.
+  ## @param tss.deployment.strategy.type Specifies the deployment strategy, RollingUpdate is preferred for minimal downtime.
+  ## @param tss.deployment.strategy.rollingUpdate.maxUnavailable The maximum number of pods that can be unavailable during an update.
+  ## @param tss.deployment.strategy.rollingUpdate.maxSurge The maximum number of pods that can be created over the desired number of pods.
+  ## @param tss.deployment.podSecurityContext Security settings for the pods.
+  ## @param tss.deployment.securityContext Security settings for the container within the pod.
+  ## @param tss.deployment.resources Resource limits and requests for the application pods.
+  ## @param tss.deployment.resources Resource limits and requests for the application pods.
```

These lines are repeated, one would suffice:
```suggestion
  ## @param tss.deployment.resources Resource limits and requests for the application pods.
```